### DISCUSSION TITLE
The Argument Double quotes not recognized

### DISCUSSION THREAD

**Discussion by HillCat** [Tyrrrz/CliWrap#193] [Q&A] [answered]
Instead of outputting the double quotes as I expected, it output all the symbols as they are, resulting in an error on the command line.
![rider64_OJvtD3I6Lt](https://user-images.githubusercontent.com/41800196/216480651-2e564b69-b959-47da-ba2c-932956798b54.png)
```
var cmd = Cli.Wrap(PythonVenvPath+"activate.bat");
            var result = await cmd.ExecuteAsync();
            var downloadSubtitle =await Cli.Wrap("whisperSubs")
                .WithArguments( new []
                {
                    "\"F:\\Downloads\\Video\\OSS_Power_Ups__CliWrap.mp4\"",
                    "--output_dir",
                    @"F:\Downloads\Video"
                    
                }).ExecuteAsync();
            Console.WriteLine();
            var saaa = string.Empty;
```

![r29tot11R3](https://user-images.githubusercontent.com/41800196/216487163-f51433a7-6be5-4ae8-8d6c-f482fc26157a.png)

If it is double quotes plus spaces, then the double quotes will be parsed out, if not, they will not be parsed out.It's really very strange.

**Comment by HillCat** [comment, marked as answer]
```
var downloadSubtitle = Cli.Wrap("whisperSubs")
                .WithArguments( new []
                {
                    "\"F:\\Downloads\\Video\\OSS_Power_Ups__CliWrap.mp4\"",
                    "--output_dir",
                    "F:\\Downloads\\Video"
                    
                },false);
```
Add the escape character, then Escape=false and you're done!

**Reply by Tyrrrz** [reply]
I'm not sure what you're trying to achieve there, but you don't need to put quotes there, CliWrap does that automatically (and in a smarter way).

Your code is equivalent to this:

```csharp
var downloadSubtitle = Cli.Wrap("whisperSubs")
	.WithArguments( new []
	{
	    "F:\\Downloads\\Video\\OSS_Power_Ups__CliWrap.mp4",
	    "--output_dir",
	    "F:\\Downloads\\Video"
	    
	});
```

Note that if CliWrap doesn't add quotes to your arguments, that means they're not necessary (i.e. there are no special characters in the argument to escape).
